client/apollo/css/src/Heading/HeadingApollo.scss

```
@@ -0,0 +1,50 @@
+@use "../common/breakpoints";
+@use "./HeadingCommon";
+
+.af-heading {
+  --texts-column: 1;
+  --heading-title-color: var(--axa-blue-100);
+  --heading-subtitle-color: var(--neutral-100);
+  --heading-subtitle-font-size: calc(16 / var(--font-size-base) * 1rem);
+
+  &--h1 {
```

je me demande si `> h1` ne serais pas mieux et éviterais de devoir mettre la class dans le composant.

---

~~je ne vois pas comment utiliser l'opérateur `>` en css et vue que ceci est concerne purement les styles, on se retrouve à le faire.~~
par contre on pourrai retirer effectivement les classes *--h{1,2,3} du composant et directement selectionner les balises h concernées

---

Ben en gros tu as juste à faire 
```suggestion
  > h1 {
```
Et cela permet de sélectionner tous les h1 directement enfant de `.af-heading`